frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentGrid.tsx

```diff
@@ -111,23 +113,30 @@ export const PublicEnrollmentGrid = ({
     []
   );
 
+  if (!selectedExamEvent) {
+    return null;
+  }
```

Comment:
Tää taitaa aiheuttaa sen, että näkyy tyhjä sivu kun Enrollmentin tai ExamEventin tietoja ladataan. Ehkä tän vois muuttaa vaikka jotenkin näin:

```suggestion
  if (!selectedExamEvent) {
    return (
      <Grid className="public-enrollment__grid" item>
        <LoadingProgressIndicator
          isLoading={true}
          translateCommon={translateCommon}
          displayBlock={true}
        />
      </Grid>
    );
  }
```